sdjournal/journal_test.go

```diff
@@ -343,3 +344,54 @@ func TestJournalReaderSmallReadBuffer(t *testing.T) {
 		t.Fatalf("Got unexpected message %s", got[1])
 	}
 }
+
+func TestJournalGetUniqueValues(t *testing.T) {
+	j, err := NewJournal()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer j.Close()
+
+	uniqueString := generateRandomField(20)
```

Comment:
I see, I got confused as one of the tests above does it, but for the value. Nevermind, let's keep it as is.